lib/db.js

```diff
@@ -418,20 +458,35 @@ module.exports = (
   }
 
   DB.prototype.updateSessionToken = function (token, userAgentString) {
-    log.trace({ op: 'DB.updateSessionToken', uid: token && token.uid })
-
-    if (! token.update(userAgentString)) {
+    log.trace({ op: 'REDIS.updateSessionToken', uid: token && token.uid })
```

Comment:
Like the other one, I'd keep this trace as `op: 'DB.updateSessionToken'` because that matches the name of the object this method is part of.